tests/contrib/operators/test_gcp_compute_operator.py

```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
```

Comment:
Don't need to do it for each operator, but can you add a case exercising the operation error path? Check that the exception is thrown and braces are trimmed.

Reply:
In c17ba0fd220cd92a6730b4eef2d2635a364ba077 I've added a test mocking an error from `setMachineType()` and verifying that the exception is thrown and the braces are trimmed.